DESCRIPTION

```diff
@@ -57,6 +57,7 @@ Imports:
     utils
 Suggests:
     bslib,
+    DT (>= 0.33),
```

Comment:
Would it be possible to avoid adding a new dependency. DT is a massive package. I think that vignette on ggplot is good enough

Reply:
make sense, will remove DT example

Reply:
72e33f4ae4933b1e4aeb80864a752c3e064f3e33